test/inductor/test_torchinductor.py

```diff
@@ -620,6 +620,17 @@ def fn(a, b, alpha):
 
         self.common(fn, (x, y, 2))
 
+    def test_add_complex2(self):
+        def fn(a, b):
+            c = a + b
+            d = a + b
+            return c + d
+
+        x = torch.tensor([1 + 1j, -1 + 1j, -2 + 2j, 3 - 3j, 0, 1j, 1, -1])
+        y = torch.tensor([1 + 1j, -1 + 1j, -2 + 2j, 3 - 3j, 0, 1j, 1, -1])
+
```

Comment:
use `run_and_get_code` to get the compiled source, then assert something about code.count("view")